test/gtest/uct/uct_test.cc

```diff
@@ -928,10 +928,12 @@ uct_test::entity::entity(const resource& resource, uct_md_config_t *md_config,
     }
 }
 
-void uct_test::entity::mem_alloc_host(size_t length, unsigned mem_flags,
-                                      uct_allocated_memory_t *mem) const
+void uct_test::entity::mem_alloc(size_t length, unsigned mem_flags,
+                                 uct_allocated_memory_t *mem,
+                                 ucs_memory_type_t mem_type) const
```

Comment:
memtype should be 2nd param

Reply:
But parameters with default values have to be last

Reply:
ok